portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -42,12 +42,54 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be deleted"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeDeleted {
+		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131223 CanBeDeleted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
```

Comment:
var fieldParagraphField = FormFields.getFieldName();

This line doesn't seem to be referenced anywhere. Is it needed to pass?